src/modules/ekf2/EKF/ekf.h

```diff
@@ -499,6 +499,24 @@ class Ekf final : public EstimatorInterface
 		return is_healthy;
 	}
 
+	bool measurementUpdate(VectorState &K, VectorState &H, float R, float innovation)
```

Comment:
Should we name these more descriptively or at least add a comment?

Reply:
Yes, that was just for a quick test